### PR TITLE
Mute all devices before running the DUB tests.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: tools/rdm/TestDefinitions.py [hunk 1]
```diff
@@ -182,6 +182,25 @@ def Test(self):
...
+    self.AddExpectedResults([
+      BroadcastResult(),
+      UnsupportedResult()
+    ])
+
+    self.SendDirectedDiscovery(
+        UID.AllDevices(),
+        PidStore.ROOT_DEVICE,
+        self.pid)
+
```
Comment (peternewman) [inline tools/rdm/TestDefinitions.py]: Add another blank line here to keep flake8 happy.

**Comment by peternewman** [comment]
Generally LGTM, just one comment.

File: tools/rdm/TestDefinitions.py (hunk 1 above)
Comment (nomis52) [inline tools/rdm/TestDefinitions.py]: Done

**Comment by peternewman** [comment]
LGTM

**Pull request merged**